src/lib/pages/frontsite/Documents/DocumentDetails/DocumentItems/DocumentItems.js

```diff
@@ -92,13 +93,8 @@ class DocumentItems extends Component {
 
   renderLibraries = () => {
     const { activeLibrary } = this.state;
-    const {
-      documentDetails: {
-        metadata: {
-          items: { on_shelf: onShelf },
-        },
-      },
-    } = this.props;
+    const { documentDetails } = this.props;
+    const onShelf = _get(documentDetails, 'metadata.items.on_shelf', {});
```

Comment:
Maybe I am mistaken, but didn't we said remote the `_get` here?

Reply:
You are right 👍